### ISSUE TITLE
help request: some error when setup tracing for Python application.

### ISSUE THREAD

**Issue by JacqueWong** [openobserve/openobserve#863]
hi, all

   when I setup tracing for the Python application, I got some errors：

```bash
127.0.0.1 - - [01/Jun/2023 13:37:38] "GET / HTTP/1.1" 200 -
Exception while exporting Span batch.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
                       ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/http/client.py", line 1374, in getresponse
    response.begin()
  File "/usr/lib/python3.11/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/http/client.py", line 287, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 532, in increment
    raise six.reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/six.py", line 718, in reraise
    raise value.with_traceback(tb)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
                       ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/http/client.py", line 1374, in getresponse
    response.begin()
  File "/usr/lib/python3.11/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/http/client.py", line 287, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

```

environment：
Python 3.11.2
Debian

> It happened when I followed the doc(https://openobserve.ai/docs/ingestion/traces/python/) test.

```bash
curl http://127.0.0.1:5000
Hello World!
```

> When I refresh the page(http://127.0.0.1:5000/) a couple of times to get more traces exported, The python script outputs the above error.

I confirm that the preceding steps are correct.
Can someone answer for me? I really appreciate any help you can provide.

**Comment by omkarK06** [comment]
@JacqueWong we have updated sample tracing app https://github.com/openobserve/sample-tracing-python. You can follow https://openobserve.ai/docs/ingestion/traces/python/ and refer this repository.

**Comment by JacqueWong** [comment]
> @JacqueWong we have updated sample tracing app https://github.com/openobserve/sample-tracing-python. You can follow https://openobserve.ai/docs/ingestion/traces/python/ and refer this repository.

thank you for your reply! I tried the new "sample tracing app". Unfortunately, the problem still exists, but I have a new discovery: when I click the "Reset token" button on the "/web/ingestion/traces/opentelemetry" page and update the new token to "tracing.py" After that, I can see the data flow increase on the "/web/logstreams" page, and see the relevant data on the "/web/traces" page, without the above errors. I have carried out the same modification test on both the window and Linux systems, both of which need to reset the token. The problem seems to have a temporary solution. Thanks again!
